.github/workflows/e2e.yml

```diff
@@ -65,9 +65,9 @@ jobs:
       - name: Install Playwright and browsers
         run: pnpm playwright install --with-deps
 
-      - name: Run sources
+      - name: Run setup
```

Comment:
I'd rather not modify all the `e2e` projects to have a `setup` script.

Instead, can you add another step to this workflow to run `setup` if it exists?

```sh
pnpm run --if-present setup
```